src/lib/VirtualScroller.js

```diff
@@ -204,12 +204,24 @@ class VirtualScroller {
      * @return {Object} - info object
      */
     getCurrentListInfo() {
-        const { firstElementChild, lastElementChild } = this.listEl;
+        const { firstElementChild, lastElementChild, children } = this.listEl;
 
-        const curStartOffset = firstElementChild ? Number.parseInt(firstElementChild.dataset.bpVsRowIndex, 10) : -1;
-        const curEndOffset = lastElementChild ? Number.parseInt(lastElementChild.dataset.bpVsRowIndex, 10) : -1;
+        // Parse the row index from the data-attribute
+        let curStartOffset = firstElementChild ? Number.parseInt(firstElementChild.dataset.bpVsRowIndex, 10) : -1;
```

Comment:
Might want to double check support for dataset on Safari. MDN has it as `?`. https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset